src/models/expenses.js

```diff
@@ -0,0 +1,33 @@
+const { DataTypes } = require("sequelize");
+const sequelize = require("../utils/db.js");
+
+const Expenses = sequelize.define('Expenses', {

```

Comment:
also here you must to set up id and make assign dependency on user table 
![image](https://github.com/mate-academy/node_accounting-app-with-db/assets/96265844/148f3921-1da6-4dee-8c58-358d3494d7c2)